database/factories/LicenseFactory.php

```diff
@@ -37,18 +38,22 @@ class LicenseFactory extends Factory
      */
     public function definition()
     {
+
+
         return [
             'user_id' => 1,
-            'license_name' => $this->faker->name,
+            'name' => $this->faker->name,
```

Comment:
Huh. Is this something that we had blown for a while? Ugh. Definitely possible though.

Reply:
The only reason I found it was because I tried to do use the Factory directly instead of the 4 licenses that you use in your normal seeder.